libs/s3/s3_cmd_test_lib.py

```diff
@@ -139,7 +138,7 @@ def download_bucket_cli(
             LOGGER.error("Error in %s: %s",
                          S3CmdTestLib.download_bucket_cli.__name__,
                          error)
-            raise CTException(err.S3_CLIENT_ERROR, error.args[0])
+            raise CTException(err.S3_CLIENT_ERROR, error.args[0]) from error
```

Comment:
Please verify changes by executing one of the +/- ve tests scenario.

Reply:
Verified